apps/openmw/mwsound/openal_output.cpp

```diff
@@ -622,6 +622,7 @@ void OpenAL_Output::init(const std::string &devname)
     }
 
     ALC.EXT_EFX = !!alcIsExtensionPresent(mDevice, "ALC_EXT_EFX");
+    AL.SOFT_source_spatialize = !!alIsExtensionPresent("AL_SOFT_source_spatialize");
```

Comment:
The !! stopped me for a second... that's a funny way to write ```static_cast<bool>```

Anyway great change. This fixes the overly-loud riekling scream sounds, doesn't it?

Reply:
The ```!!``` is something I picked up in earlier days using C (which didn't have an explicit bool type readily available). And it just seems more lean and elegant to me than ```static_cast<bool>(...)```. If it's an issue, I can actually change the flags to bools for the cast to happen automatically (I just wasn't sure if the size specifier worked properly with bools, but they do).

It does fix the riekling sounds, yes. ```cr/riek/moan.wav``` is stereo for some reason (other sounds may be too), and this will properly handle it.